### ISSUE TITLE
[UI] New notifaction if some functionality is not supported on a chain

### ISSUE THREAD

**Issue by holkexyz** [hypercerts-org/hypercerts-app#300]
On Filecoin Calibration the marketplace features are not supported yet. When I click on marketplace on my profile I receive an error (see below). Instead of this error, I just want to receive a message: "This functionality is not yet supported on this chain."

Alternatively, the marketplace button should only be shown if it is supported.

**Comment by bitbeckers** [comment]
@holkexyz I don't see an error, also not in the app. Listing not available on the chain are grayed out - maybe washed out, but you get the point

<img width="1402" alt="Image" src="https://github.com/user-attachments/assets/0720f390-0363-4a3b-a9f5-b5a387b85e72" />

**Comment by holkexyz** [comment]
Yes, I also don't get an error anymore. And the guess I missed to include the screenshot in the original message.

So, this is great:

<img width="1276" alt="Image" src="https://github.com/user-attachments/assets/9361f6b0-d760-46e2-9ffa-4bc932ef84dc" />
